public/app/percona/add-instance/components/AddRemoteInstance/FormParts/PostgreSQLConnectionDetails/PostgreSQLConnectionDetails.tsx

```diff
@@ -83,6 +83,14 @@ export const PostgreSQLConnectionDetails: FC<MainDetailsFormPartProps> = ({ form
           tooltipText={Messages.form.tooltips.postgresqlDetails.maxQueryLength}
         />
       </div>
+      <div className={styles.group}>
```

Comment:
I think you are missing here the check for an azure instance. Otherwise looks good

Reply:
Updated